node/runtime/pangolin/src/pallets/evm.rs

```diff
@@ -146,22 +146,23 @@ where
 			a if a == addr(7) => Some(Bn128Mul::execute(input, target_gas, context, is_static)),
 			a if a == addr(8) => Some(Bn128Pairing::execute(input, target_gas, context, is_static)),
 			a if a == addr(9) => Some(Blake2F::execute(input, target_gas, context, is_static)),
-			// Darwinia precompiles: 1024+ for stable precompiles, 2048+ for experimental
-			// precompiles
+			// Darwinia precompiles: 1024+ for stable precompiles.
+			// TODO: Change the transfer precompile address after https://github.com/darwinia-network/darwinia-common/issues/1259
 			a if a == addr(21) =>
 				Some(<Transfer<R>>::execute(input, target_gas, context, is_static)),
+			// TODO: Delete EthereumBridge and Sub2SubBridge precompiles in the futures.
 			a if a == addr(23) =>
 				Some(<EthereumBridge<R>>::execute(input, target_gas, context, is_static)),
 			a if a == addr(24) => Some(<Sub2SubBridge<
 				R,
 				ToPangoroMessageSender,
 				bm_pangoro::ToPangoroOutboundPayLoad,
 			>>::execute(input, target_gas, context, is_static)),
-			a if a == addr(25) =>
```

Comment:
@xiaoch05 Are you using this precompile in helix on testnets?

And are you ok with removing it if you will need to re-deploy contracts or you would rather to keep this id too?

Reply:
ok, I will try to upgrade the contract on testnets after this update merged into pangolin network.

Reply:
Please review again.